==================
WARNING: ThreadSanitizer: data race (pid=29059)
  Atomic read of size 1 at 0x7b6800002300 by main thread:
    #0 pthread_mutex_lock /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/../sanitizer_common/sanitizer_common_interceptors.inc:3914:3 (myApp+0x5d26e)
    #1 <null> <null> (libomp.so.5+0x739be)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Previous write of size 1 at 0x7b6800002300 by thread T3:
    #0 pthread_mutex_init /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:1211:3 (myApp+0x43033)
    #1 <null> <null> (libomp.so.5+0x71661)

  Location is heap block of size 1440 at 0x7b6800001e00 allocated by main thread:
    #0 malloc /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:666:5 (myApp+0x40457)
    #1 <null> <null> (libomp.so.5+0x1d4a1)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Thread T3 (tid=29063, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

SUMMARY: ThreadSanitizer: data race (/usr/lib/x86_64-linux-gnu/libomp.so.5+0x739be) 
==================
LLVMSymbolizer: error reading file: No such file or directory
==================
WARNING: ThreadSanitizer: data race (pid=29059)
  Write of size 8 at 0x7ffeea1f5b98 by main thread:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 (myApp+0xa909d)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)
    #3 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Previous read of size 8 at 0x7ffeea1f5b98 by thread T1:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:15 (myApp+0xa903e)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Location is stack of main thread.

  Location is global '??' at 0x7ffeea1e7000 ([stack]+0x00000000eb98)

  Thread T1 (tid=29061, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

SUMMARY: ThreadSanitizer: data race /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 in .omp_outlined._debug__
==================
==================
WARNING: ThreadSanitizer: data race (pid=29059)
  Write of size 8 at 0x7ffeea201718 by thread T3:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 (myApp+0xa909d)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Previous read of size 8 at 0x7ffeea201718 by thread T4:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:15 (myApp+0xa903e)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Location is stack of main thread.

  Location is global '??' at 0x7ffeea1e7000 ([stack]+0x00000001a718)

  Thread T3 (tid=29063, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Thread T4 (tid=29064, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

SUMMARY: ThreadSanitizer: data race /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 in .omp_outlined._debug__
==================
==================
WARNING: ThreadSanitizer: data race (pid=29059)
  Write of size 8 at 0x7ffeea1f9a18 by thread T1:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 (myApp+0xa909d)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Previous read of size 8 at 0x7ffeea1f9a18 by thread T2:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:15 (myApp+0xa903e)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Location is stack of main thread.

  Location is global '??' at 0x7ffeea1e7000 ([stack]+0x000000012a18)

  Thread T1 (tid=29061, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Thread T2 (tid=29062, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

SUMMARY: ThreadSanitizer: data race /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 in .omp_outlined._debug__
==================
==================
WARNING: ThreadSanitizer: data race (pid=29059)
  Write of size 8 at 0x7ffeea1fd898 by thread T2:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 (myApp+0xa909d)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Previous read of size 8 at 0x7ffeea1fd898 by thread T3:
    #0 .omp_outlined._debug__ /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:15 (myApp+0xa903e)
    #1 .omp_outlined. /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:73:3 (myApp+0xa91dc)
    #2 __kmp_invoke_microtask <null> (libomp.so.5+0x7c452)

  Location is stack of main thread.

  Location is global '??' at 0x7ffeea1e7000 ([stack]+0x000000016898)

  Thread T2 (tid=29062, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

  Thread T3 (tid=29063, running) created by main thread at:
    #0 pthread_create /home/yshi10/ARCHER/ArcherBuild/llvm_src/projects/compiler-rt/lib/tsan/rtl/tsan_interceptors.cc:992:3 (myApp+0x41f25)
    #1 <null> <null> (libomp.so.5+0x72110)
    #2 __libc_start_main /build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:310 (libc.so.6+0x21b96)

SUMMARY: ThreadSanitizer: data race /home/yshi10/datarace/RaceDetectionService/tools_output/dataRaceTest8/DRB014-outofbounds-orig-yes.c:75:14 in .omp_outlined._debug__
==================
b[50][50]=0.000000
ThreadSanitizer: reported 5 warnings

